packaging/windows/build_pypi_pkg_download_proxy/setup.py

```diff
@@ -14,18 +14,23 @@
 
 install_requires = [
    'pip',
+   'future',
```

Comment:
`future` shouldn't be needed since we are only targeting Python-3.6+, aren't we?
```suggestion
```

Reply:
This unfortunately is required in the wheel so I had to keep it here.